YaLibs/YaToolsIDALib/Repository.cpp

```diff
@@ -750,20 +769,34 @@ bool Repository::rebase(const std::string& upstream, const std::string& destinat
 
 bool Repository::add_file_to_index(const std::string& path)
 {
+    fs::path p = path;
+    if (!include_idb_ &&
+        (path == get_current_idb_name() ||
+        path == get_original_idb_name())) {
+        p.replace_extension(".idx_file");
+        std::fstream f;
+        f.open(p, std::fstream::out);
+        f << "DO NOT DELETE!" << std::endl;
+    }
     try
     {
-        repo_.add_file(path);
+        repo_.add_file(p);
     }
     catch (const std::runtime_error& error)
     {
-        LOG(WARNING, "Failed to add %s to index, error: %s\n", path.c_str(), error.what());
+        LOG(WARNING, "Failed to add %s to index, error: %s\n", p.c_str(), error.what());
```

Comment:
same here, p.c_str() does not exist, you need to convert to std::string first with p.generic_string().c_str()

Reply:
fixed in f4e387aa68d432c5dc6030e0f89dc6e71f0760e2